apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { PropTypes } from 'react';
+import Radium from 'radium';
+import color from "@cdo/apps/util/color";
+
+export const DOT_SIZE = 30;
+
+// TODO - not sure I like these being keyed by these strings.
```

Comment:
Enum def feels justified. I was wondering a little bit if I'd rather send in a result type (i.e. a number like 100) and then let the component own mapping that to a set of styles. There's weirdness right now in that the server sends down essentially a CSS class name, and then we use that to toggle all sorts of things. I'd like to clean that up at some point, but I think to start with it's reasonable to just make this into an enum.